src/entities.py

```diff
@@ -226,63 +226,64 @@ def to_api_representation(self) -> Dict[str, object]:
-    CELERY = {Language.DE: "Sellerie"}
-    MUSTARD = {Language.DE: "Senf"}
-    SESAME = {Language.DE: "Sesam"}
-    SULPHURS = {Language.DE: "Schwefeldioxid"}
-    SULFITES = {Language.DE: "Sulfite"}
-    LUPIN = {Language.DE: "Lupine"}
-    MOLLUSCS = {Language.DE: "Weichtiere"}
-    SHELL_FRUITS = {Language.DE: "Schalenfrüchte"}
-
-    BAVARIA = {Language.DE: "Zertifizierte Qualität Bayern"}
-    MSC = {Language.DE: "Marine Stewardship Council"}
-    DYESTUFF = {Language.DE: "Farbstoffe"}
-    PRESERVATIVES = {Language.DE: "Preservate"}
-    ANTIOXIDANTS = {Language.DE: "Antioxidanten"}
-    FLAVOR_ENHANCER = {Language.DE: "Geschmacksverstärker"}
-    WAXED = {Language.DE: "Gewachst"}
-    PHOSPATES = {Language.DE: "Phosphate"}
-    SWEETENERS = {Language.DE: "Süßungsmittel"}
-    PHENYLALANINE = {Language.DE: "Phenylaline"}
-    COCOA_CONTAINING_GREASE = {Language.DE: "Kakaohaltiges Fett"}
-    GELATIN = {Language.DE: "Gelatine"}
-    ALCOHOL = {Language.DE: "Alkohol"}
-    PORK = {Language.DE: "Schweinefleisch"}
-    BEEF = {Language.DE: "Rinderfleisch"}
-    VEAL = {Language.DE: "Kalbsfleisch"}
-    WILD_MEAT = {Language.DE: "Wildfleisch"}
-    LAMB = {Language.DE: "Lammfleisch"}
-    GARLIC = {Language.DE: "Knoblauch"}
-    POULTRY = {Language.DE: "Geflügel"}
-    CEREAL = {Language.DE: "Getreide"}
-    MEAT = {Language.DE: "Fleisch"}
-    VEGAN = {Language.DE: "Vegan"}
-    VEGETARIAN = {Language.DE: "Vegetarisch"}
+        self.abbreviation = abbreviation
+
+    GLUTEN = {Language.DE: "Gluten"}, "🌾"
+    WHEAT = {Language.DE: "Weizen"}, None
+    RYE = {Language.DE: "Roggen"}, None
+    BARLEY = {Language.DE: "Gerste"}, None
+    OAT = {Language.DE: "Hafer"}, None
+    SPELT = {Language.DE: "Dinkel"}, None
+    HYBRIDS = {Language.DE: "Hybridstämme"}, None
+    SHELLFISH = {Language.DE: "Krebstiere"}, "🦀"
+    CHICKEN_EGGS = {Language.DE: "Eier"}, "🥚"
+    FISH = {Language.DE: "Fisch"}, "🐟"
+    PEANUTS = {Language.DE: "Erdnüsse"}, "🥜"
+    SOY = {Language.DE: "Soja"}, None
+    MILK = {Language.DE: "Milch"}, "🥛"
+    LACTOSE = {Language.DE: "Laktose"}, None
+    ALMONDS = {Language.DE: "Mandeln"}, None
+    HAZELNUTS = {Language.DE: "Haselnüsse"}, "🌰"
+    WALNUTS = {Language.DE: "Walnüsse"}, None
+    CASHEWS = {Language.DE: "Cashewnüsse"}, None
+    PECAN = {Language.DE: "Pekanüsse"}, None
+    PISTACHIOES = {Language.DE: "Pistazien"}, None
+    MACADAMIA = {Language.DE: "Macadamianüsse"}, None
+    CELERY = {Language.DE: "Sellerie"}, None
+    MUSTARD = {Language.DE: "Senf"}, None
+    SESAME = {Language.DE: "Sesam"}, None
+    SULPHURS = {Language.DE: "Schwefeldioxid"}, "🔶"
+    SULFITES = {Language.DE: "Sulfite"}, "🔻"
+    LUPIN = {Language.DE: "Lupine"}, None
+    MOLLUSCS = {Language.DE: "Weichtiere"}, "🐙"
+    SHELL_FRUITS = {Language.DE: "Schalenfrüchte"}, None
+
+    BAVARIA = {Language.DE: "Zertifizierte Qualität Bayern"}, "GQB"
+    MSC = {Language.DE: "Marine Stewardship Council"}, "MSC"
```

Comment:
How about 🎣 instead of MSC?
```suggestion
    MSC = {Language.DE: "Marine Stewardship Council"}, "🎣"
```

Reply:
Sounds good